src/otx/engine/engine.py

```diff
@@ -240,6 +245,9 @@ def train(
         """
         checkpoint = checkpoint if checkpoint is not None else self.checkpoint
 
+        if adaptive_bs != "None":
+            adapt_batch_size(engine=self, **locals(), not_increase=(adaptive_bs != "Full"))
```

Comment:
If `adaptive_bs` and `run_hpo` are running at the same time, will this cause any problems or do we need to prevent this?

Reply:
HPO is turned off during adaptive batch size and adaptive batch size is turned of during HPO.